### DISCUSSION TITLE
Model Context Protocol

### DISCUSSION THREAD

**Discussion by S1M0N38** [olimorris/codecompanion.nvim#559] [Ideas]
Is there any plan to integrate `codecompanion.nvim` with the [Model Context Protocol](https://modelcontextprotocol.io/introduction)?

If `codecompanion.nvim` implements a compatible MCP client interface, it could leverage a variety of [MCP servers](https://github.com/modelcontextprotocol/servers) to provide additional context to LLMs from diverse sources.

I’m curious to hear your thoughts on this.

**Comment by andtankian** [comment]
It would be a great feature, actually it would just make this plugin out of this world!

**Reply by olimorris** [reply]
See #1058. It's incredible!

**Comment by olimorris** [comment]
Closing this as #1058 has taken care of this
